android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -229,6 +229,13 @@ object WebExtensionSupport {
 
                 override fun onInstalled(extension: WebExtension) {
                     registerInstalledExtension(store, extension)
+                    if (!extension.isBuiltIn()) {
```

Comment:
I think we should add a comment here to explain why we filter out built-in extensions.

Reply:
Added!